cesium/build_model.py

```diff
@@ -71,7 +83,38 @@ def fit_model_optimize_hyperparams(data, targets, model, params_to_optimize,
 def build_model_from_featureset(featureset, model=None, model_type=None,
                                 model_options={}, params_to_optimize=None,
                                 cv=None):
-    """Build model from (non-rectangular) xarray.Dataset of features."""
+    """Build model from (non-rectangular) xarray.Dataset of features.
+
+    Parameters
+    ----------
+    featureset : xarray.Dataset of features
+        Features for training model.
+    model : scikit-learn model, optional
+        Instantiated scikit-learn model. If None, `model_type` must not be.
+        Defaults to None.
+    model_type : str, optional
+        String indicating model to be used, e.g. "RandomForestClassifier".
+        If None, `model` must not be. Defaults to None.
+    model_options : dict, optional
+        Dictionary with hyperparameter values to be used in model building.
+        Keys are parameter names, values are the associated values.
```

Comment:
I don't know why this is called `model_options`, so how about something along the lines of (please adjust as necessary):

```
model_parameters : dict, optional
    During hyper-parameter optimization, various model parameters are adjusted to give an 
    optimal fit.  This dictionary gives the different values that should be explored for each
    parameter.  E.g., `{'alpha': [1, 2], 'beta': [4, 5, 6]}` would fit models on all
    6 combinations of alpha and beta and compare the resulting models'
    goodness-of-fit.
```